consumer/src/main/kotlin/streams/procedures/StreamsSinkProcedures.kt

```diff
@@ -35,6 +48,80 @@ class StreamsSinkProcedures {
         return data.map { StreamResult(mapOf("data" to it)) }.stream()
     }
 
+    @Procedure("streams.sink.start")
+    fun sinkStart(): Stream<StreamsSinkDTO> {
+        checkEnabled()
+        checkLeader()
+        try {
+            streamsEventSink?.start()
+            return sinkStatus()
+        } catch (e: Exception) {
+            log?.error("Cannot start the Sink because of the following exception", e)
+            return Stream.concat(sinkStatus(),
+                    Stream.of(StreamsSinkDTO("exception", ExceptionUtils.getStackTrace(e))))
+        }
+    }
+
+    @Procedure("streams.sink.stop")
+    fun sinkStop(): Stream<StreamsSinkDTO> {
+        checkEnabled()
+        checkLeader()
+        try {
+            streamsEventSink?.stop()
```

Comment:
what happens with the status that is managed by the kernel-extension-factory?

Will it try to stop / start the sink again if it's already started or stopped? Can we make sure that still works.

.e.g. someone calls sink.stop and then the database triggers a stop, will that be handled gracefully

or a restart of the db in the cluster

Reply:
Great question! I'll manage this in the next PR where I'll add all the changes made in #344